tests/test_utils/test_flash_attention.py

```diff
@@ -0,0 +1,86 @@
+import torch
+import pytest
+from einops import rearrange
+from colossalai.kernel.cuda_native.flash_attention import flash_attention, triton_flash_attention, triton_check
+
+
+TRITON_AVALIABLE = triton_check()
```

Comment:
from colossalai.kernel.cuda_native.flash_attention import TRITON_AVALIABLE
do not need to run it again?